sources/System/Renderer/RenderSystem.cpp

```diff
@@ -20,8 +20,11 @@ namespace BBM
 		}),
 		_wal(wal),
 		_window(window),
-		_camera(Vector3f(), Vector3f(), Vector3f(0, 1, 0), 50, CAMERA_PERSPECTIVE)
-	{}
+		_camera(Vector3f(), Vector3f(), Vector3f(0, 1, 0), 50, CAMERA_PERSPECTIVE),
+		_debugMode(true)
```

Comment:
you should take it as param

Reply:
debug could be a friend class which then let it access to private date in your engine